source/_integrations/binary_sensor.rest.markdown

```diff
@@ -84,7 +84,7 @@ name:
   description: Name of the REST binary sensor.
   required: false
   type: template
-  default: REST Binary Sensor
+  default: REST binary sensor
```

Comment:
```suggestion
  default: REST Binary Sensor
```

This describes the default value set by the integration, so unless the integration has been changed in core, this should not change.